GDJS/GDJS/Extensions/Builtin/Capacities/EffectExtension.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * GDevelop JS Platform
+ * Copyright 2008-2023 Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#include "GDJS/Extensions/Builtin/Capacities/EffectExtension.h"
+#include "GDCore/Extensions/Builtin/AllBuiltinExtensions.h"
+#include "GDCore/Extensions/Metadata/InstructionMetadata.h"
+#include "GDCore/Tools/Localization.h"
+
+namespace gdjs {
+
+EffectExtension::EffectExtension() {
+  gd::BuiltinExtensionsImplementer::ImplementsEffectExtension(*this);
+
+  GetBehaviorMetadata("EffectCapability::EffectBehavior")
+      .SetIncludeFile("object-capabilities/EffectBehavior.js");
+
+  auto& actions = GetAllActionsForBehavior("EffectCapability::EffectBehavior");
+  auto& conditions = GetAllConditionsForBehavior("EffectCapability::EffectBehavior");
+  auto& expressions = GetAllExpressionsForBehavior("EffectCapability::EffectBehavior");
+  auto& strExpressions = GetAllStrExpressionsForBehavior("EffectCapability::EffectBehavior");
```

Comment:
You can remove both expressions and strExpressions as it seems like you don't use them below